src/diffusers/schedulers/scheduling_pndm.py

```diff
@@ -110,7 +110,9 @@ def set_timesteps(self, num_inference_steps, offset=0):
             # produce better results. When using PNDM with `self.config.skip_prk_steps` the implementation
             # is based on crowsonkb's PLMS sampler implementation: https://github.com/CompVis/latent-diffusion/pull/51
             self.prk_timesteps = np.array([])
-            self.plms_timesteps = (self._timesteps[:-1] + self._timesteps[-2:-1] + self._timesteps[-1:])[::-1].copy()
```

Comment:
this adds numbers in numpy, doesn't concatenate. Solves issues from PR: https://github.com/huggingface/diffusers/commit/b64c5227595f5eed10f6ff3ac7953de0bb07ab2d